assets/styles/style.css

```diff
@@ -786,17 +808,483 @@ footer {
+    line-height: 33px;
+    letter-spacing: 0em;
+    padding-top: 40px;
+}
+
+.image-container {
+    margin-top: 40px;
+}
+
+.guests-container-item {
+    white-space: nowrap;
+}
+
+.slide-item-img>img {
+    height: 156px;
+    width: 156px;
+}
+
+.slide-item-first>img {
+    height: 156px;
+    width: 156px;
+}
+
+.slide-item-title {
+    font-family: Roboto;
+    font-size: 28px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 33px;
+    letter-spacing: 0em;
+}
+
+.slide-item-text {
+    font-family: Roboto;
+    font-size: 28px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 33px;
+    letter-spacing: 0em;
+}
+
+.image-item-name {
+    font-family: Roboto;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0em;
+    margin: 10px 0;
+}
+
+.guests-container-item>p {
+    font-family: Roboto;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0em;
+    text-align: left;
+    
+}
+
+/*   destination media   */
+
+.content-section {
+    width: 100%;
+}
+
+.destimation--section {
+    height: 379px;
+}
+
+.next-content-page {
+    display: none;
+}
+
+.navi-item {
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+}
+
+.destination-section-title {
+    font-family: Roboto;
+    font-size: 28px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 33px;
+    letter-spacing: 0em; 
+    color: #383838;
+    text-align: center;
+    padding-top: 40px;
+    margin-bottom: 40px;
+}
+
+.destination-nav-item {
+    height: 32px;
+    width: 120px;
+    border-radius: 4px; 
+    border-radius: 8px;
+    display: grid;
+    place-items: center;
+    white-space: nowrap;
+    margin-right: 25px;
+}
+
+.destination-nav-item>a {
+    text-decoration: none;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 19px;
+    line-height: 22px;
+    font-weight: 400;
+    letter-spacing: 0em;
+    text-align: center;
+    color: #383838;
+}
 
+.destination-nav-item>a:hover {
+    color: #ffffff;
+    height: 40px;
+    width: 140px;
+    display: grid;
+    place-items: center;
+    background: #3077C6;
+    border-radius: 8px;
+}
+
+.destination--navigation {
+    margin-bottom: 25px;
+}
+
+/*       sign-up media                           */
+
+
+
+.sign-up-section {
+    display: none;
+}
+
+/*       reviews media                           */
+
+.reviews-container {
+    width: 100%;
+    height: 800px;
+}
+
+.guest-reviews-section-title {
+    display: none;
+}
+
+.guest-reviews-media-title {
+    display: flex;
+    font-family: Roboto;
+    font-size: 28px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 33px;
+    letter-spacing: 0em;
+    justify-content: center;
+    padding-top: 40px;
+    margin-bottom: 40px;
+
+}
+
+.reviews--section-items {
+    display: flex;
+    justify-content: space-evenly;
+}
```

Comment:
Check distance between blocks for medium displays
![image](https://user-images.githubusercontent.com/20189313/129494091-87ab783e-639a-4cae-8a04-43de3ba99cbd.png)